### PR TITLE
Added check in hello sample

### PR DESCRIPTION
**Pull request by antonrom23**
[39942] , Added check for output size. Sample support only one output. Fixed crash, when we use  models with more than one output.

### CODE REVIEW

File: inference-engine/samples/hello_classification/main.cpp [hunk 1]
```diff
@@ -130,6 +130,7 @@ int main(int argc, char *argv[]) {
 
         // --------------------------- 8. Process output ------------------------------------------------------
         Blob::Ptr output = infer_request.GetBlob(output_name);
+        if (output->size() != 1) throw std::logic_error("Sample supports topologies with 1 output only");
```
Comment (ilya-lavrenov) [inline inference-engine/samples/hello_classification/main.cpp:133]: I suppose `output->size()` defines a number of classes, is not it?

Comment (ilyachur) [reply inference-engine/samples/hello_classification/main.cpp:133]: Yes I agree that check is incorrect.

If you want to check that topology has only one output you need to check the number of outputs. At the current moment you check the size of output blob.

Comment (antonrom23) [reply inference-engine/samples/hello_classification/main.cpp]: Yes, sorry, this check is incorrect. I updated check. @ilya-lavrenov , @ilyachur  can you look at new version?

File: inference-engine/samples/hello_classification/main.cpp [hunk 2]
```diff
@@ -130,6 +130,7 @@ int main(int argc, char *argv[]) {
 
         // --------------------------- 8. Process output ------------------------------------------------------
         Blob::Ptr output = infer_request.GetBlob(output_name);
+        if (network.getOutputsInfo().size() != 1) throw std::logic_error("Sample supports topologies with 1 output only");
```
Comment (mryzhov) [inline inference-engine/samples/hello_classification/main.cpp:133]: It is better to perform such a check right after the reading of the model:  
CNNNetwork network = ie.ReadNetwork(input_model);

**Review by mryzhov — CHANGES_REQUESTED** [review]

**Review by mryzhov — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**